cmds/zui/prov.go

```diff
@@ -43,13 +40,47 @@ func provisionRender(client zbus.Client, grid *ui.Grid, render *Flag) error {
 	go func() {
 		for counter := range counters {
 			rows := prov.Rows
-			rows[0][1] = fmt.Sprint(counter.Container)
-			rows[1][1] = fmt.Sprint(counter.Volume)
+			rows[1][1] = fmt.Sprint(counter.Container)
+			rows[1][3] = fmt.Sprint(counter.Volume)
 			rows[2][1] = fmt.Sprint(counter.Network)
-			rows[3][1] = fmt.Sprint(counter.VM)
-			rows[4][1] = fmt.Sprint(counter.ZDB)
-			rows[5][1] = fmt.Sprint(counter.Debug)
+			rows[2][3] = fmt.Sprint(counter.VM)
+			rows[3][1] = fmt.Sprint(counter.ZDB)
+			rows[3][3] = fmt.Sprint(counter.Debug)
+
+			render.Signal()
+		}
+	}()
 
+	sysMonitor := stubs.NewSystemMonitorStub(client)
+	stream, err := sysMonitor.CPU(context.Background())
+	if err != nil {
+		return errors.Wrap(err, "failed to start cpu monitor stream")
+	}
+
+	go func() {
+		var aggregatedValue float64
+		for point := range stream {
+			aggregatedValue = 0
+			for _, cpu := range point {
+				aggregatedValue += cpu.Percent
```

Comment:
May you can't ADD percentage like that :D the system monitor uses psutils with some tools to return an avg system usage

Reply:
erm how and where then? `cpu.Percent` can be a value like for example `2` or `25`, you can add integers and get an average of them as far as I remember...